### PR TITLE
Add Team Rep page for handbook

### LINKED ISSUE #6: Add team rep informational page
Team Rep responsibilities are outlined in the official [Team Reps post](https://make.wordpress.org/updates/team-reps/), but information specific to Test Team is communicated manually from rep to rep each year. The responsibilities unique to the team should be documented.

Some role/responsibility information can be gathered from recent rep nomination posts:
- https://make.wordpress.org/test/2022/08/10/test-team-reps-call-for-nominations-2/
- https://make.wordpress.org/test/2021/07/20/test-team-reps-call-for-nominations/

An example team rep page from the Design Team: https://make.wordpress.org/design/handbook/about-the-team/team-reps/

### PR DESCRIPTION
**Pull request by ironprogrammer**
Adds Test Team Rep informational page to handbook. Content adapted from recent call for nominations posts (referenced in #6).

Fixes #6.

### CODE REVIEW

File: team-reps.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Test Team Reps
+
+For a quick refresher of Team Rep roles across the project, please see the [official Team Reps post on Team Updates](https://make.wordpress.org/updates/team-reps/).
+
+## Current Reps
+
+The Test Team Reps for the 2022-2023 term are @boniu91 and @ironprogrammer.
```
Comment (ironprogrammer) [inline team-reps.md:7]:
```suggestion
The Test Team Reps for the 2023-2024 term are @webtechpooja and @ankit-k-gupta.
```
Reps update following the recent [voting period](https://make.wordpress.org/test/2023/09/27/test-team-reps-submit-your-votes/).

**Comment by ankitguptaindia** [comment]
Hello @ironprogrammer Thanks for creating this PR. I reviewed this PR and looks good to me. I reviewed the content with the reference to recent post [Test Team Reps: Call for Nominations](https://make.wordpress.org/test/2023/08/15/test-team-reps-call-for-nominations-3/)
Please let me know if there is any other process team follows to review such types of handbook PR. Thank you!

**Comment by ironprogrammer** [comment]
Thanks reviewing, @ankitguptaindia! I'll "merge" and create the corresponding page in the handbook, and we can gather additional feedback ([issues](https://github.com/WordPress/test-handbook/issues)) once it's been published.

**Comment by ironprogrammer** [comment]
The new [Test Team Reps](https://make.wordpress.org/test/handbook/team-reps/) page has been published. Will merge this to close the PR.

**Pull request merged**